event_service/repository/event_repository.py

```diff
@@ -0,0 +1,123 @@
+"""Event repository with data from database."""
+from typing import Optional, List
+from datetime import datetime
+
+from prisma.models import Event as PrismaEvent
+
+from db.postgres_client import PostgresClient
+from src.models.event import Event
+from utils.singleton import singleton
+from errors.value_not_found_error import ValueNotFoundError
+
+
+@singleton
+class EventRepository:
+    """
+    Data class that stores event information.
+
+    Attributes
+    ----------
+    _db_client : prisma.Client
+        Postgres db client.
+
+    Methods
+    -------
+    async get_events(author_id)
+        Returns events that has matches with given author id.
+    async create_event(event)
+        Creates new event inside db or throws an exception.
+    async update_event(event)
+        Updates event that has the same id as provided event object inside db or throws an exception.
+    async delete_event(event_id)
+        Deletes event that has matching id from database or throws an exception.
+
+    """
+
+    _db_client: PostgresClient
+
+    def __init__(self) -> None:
+        self._db_client = PostgresClient()
+
+    async def get_events(self, author_id: str) -> List[Event]:
+        """
+        Get events by author id.
+
+        Parameters
+        ----------
+        author_id : str
+            Author's id.
+
+        Returns
+        -------
+        List[Event]
+            List of events that matches by author id.
+
+        Raises
+        ------
+        prisma.errors.PrismaError
+            Catch all for every exception raised by Prisma Client Python.
+        ValueNotFoundError
+            No events was found for given author id.
+
+        """
+        db_events: Optional[
+            List[PrismaEvent]
+        ] = await self._db_client.db.event.find_many(where={"id": author_id})
+        if db_events is None or len(db_events) == 0:
```

Comment:
It's okay, according to documentation of python https://docs.python.org/3/reference/expressions.html#boolean-operations